doc/reference/data.rst

```diff
@@ -23,13 +23,28 @@ of an XML data file is the following:
 
     <!-- the root elements of the data file -->
     <odoo>
-      <operation/>
-      ...
+        <operation/>
+        ...
     </odoo>
 
 Data files are executed sequentially, operations can only refer to the result
 of operations defined previously
 
+Odoo Tag Parameter
+------------------
+
+``noupdate``
+   ``values : 0 or 1``
+    As name suggest, ``noupdate="0"`` mode make record can be modify by other advanced modules otherwise not with ``noupdate="1"``.
```

Comment:
As the name suggests, ``noupdate="0"`` mode makes the record modifiable by other modules. ``noupdate="1"``, on the other hand, makes the record unmodifiable.

Reply:
PR Updated. Thanks!